Acornfile

```diff
@@ -52,14 +52,16 @@ secrets: admin: {
 	name:        "Admin user credentials"
 	description: "Credentials for the admin user with full control over the database server"
 	type:        "basic"
-	data: username: "root"
+	params: passwordCharacters: "A-Za-z0-9^_-"
+	data: username:             "root"
 }
 
 secrets: user: {
 	name:        "DB User Credentials"
 	description: "Credentials for the user with full access to the created database instance"
 	type:        "basic"
-	data: username: args.username
+	params: passwordCharacters: "A-Za-z0-9^_-"
+	data: username:             args.username
```

Comment:
If this is an empty string, then it will generate. Can we provide a similar character set for the randomly generated user?

Reply:
yes we can, any preference on the character set or length? I believe it follows the same rules as passwords.

Reply:
done